### PR TITLE
:pencil: OPENSSL error fix added to README

### PR DESCRIPTION
**Pull request by treneff**
- [Description](#description)

# Description

Updated README with fixes regarding OPENSSL and Node 17+ conflicts on bundling of the application

### CODE REVIEW

**Review by DougieWougie — COMMENTED** [review]
Hi @treneff I'd rephrase that a bit because the line above says to use Node 16 (which was LTS when the Boilerplate was adopted).

Recommending nvm/nvs should probably have its own point (it does on the graphical guides in Whimsical).

**Review by DougieWougie — APPROVED** [review]
Thanks @treneff!

**Review by joefhall — APPROVED** [review]
Nice one, thanks @treneff!
